app/code/Magento/Webapi/Api/Data/ClientInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-server for the canonical source repository
```

Comment:
Copying lamias components to Magento isn't a really good idea.
More tails here https://github.com/magento/magento2/pull/33373#discussion_r676573460
I still didn't get any answer.